x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_overview/service_overview.spec.ts

```diff
@@ -73,6 +127,34 @@ describe('Service Overview', () => {
     );
   });
 
+  it('renders transaction latency chart', () => {
+    cy.get('[data-test-subj="latencyChart"]');
+  });
+
+  it('renders throughput chart', () => {
+    cy.get('[data-test-subj="throughput"]');
+  });
+
+  it('renders transactions group table', () => {
+    cy.get('[data-test-subj="transactionsGroupTable"]');
+  });
+
+  it('renders error table', () => {
+    cy.get('[data-test-subj="serviceOverviewErrorsTable"]');
+  });
+
+  it('renders dependencies table', () => {
+    cy.get('[data-test-subj="dependenciesTable"]');
+  });
+
+  it('renders instances latency distribution chart', () => {
+    cy.get('[data-test-subj="instancesLatencyDistribution"]');
+  });
+
+  it('renders instances table', () => {
+    cy.get('[data-test-subj="serviceOverviewInstancesTable"]');
+  });
+
   it('hides dependency tab when RUM service', () => {
```

Comment:
It seems that now the `cy.visit(baseUrl)` is in `beforeEach`, `/app/apm/services/opbeans-node/overview` is first loaded and then `/app/apm/services/opbeans-rum/overview`